modules/hosted-che/partials/ref_about-hosted-che.adoc

```diff
@@ -7,19 +7,19 @@
 = About Eclipse Che hosted by Red Hat
 
 Eclipse Che hosted by Red Hat is an open source product based on link:https://www.eclipse.org/che/[Eclipse {prod-short}] that is running on link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated].
-The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/codeready-workspaces/overview[CodeReady Workspaces], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
+The new service is part of the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox for Red Hat OpenShift] offering, and is using link:https://developers.redhat.com/products/devspaces/overview[Red Hat OpenShift Dev Spaces], which is built upon Eclipse Che and is optimized for Red Hat OpenShift and Red Hat Linux.
 
-== Differences between Eclipse Che and CodeReady Workspaces
+== Differences between Eclipse Che and Red Hat OpenShift Dev Spaces
```

Comment:
Why not use the `{prod-short}` name here? 

Why are you hardcoding everything when we have variables for this purpose?

Reply:
new URL will be https://developers.redhat.com/products/openshift_dev_spaces/overview - see https://issues.redhat.com/browse/CRW-2837

Reply:
Because I'm new to the project and didn't know how to find out which attribute to use^^ Thank you, Nick!

Reply:
I mean, it used to be hardcoded before so I went with the same solution, but I'll try to set up an attribute or find out whether there is one somewhere already.